### PR TITLE
Mudskipper-class Salvage Clipper

### PR DESCRIPTION
**Pull request by goober3**
## About The Pull Request
mudskipper-2
image
mudskipper-area-2

adds the mudskipper-class salvage clipper, an independent replacement to the scav-class. the mudskipper is a small, budget scavenger vessel. housing only 4 crewmembers, it specializes heavily in ruin-raiding & salvage, at the cost of lacking in every other department. it also boasts a much lower depressurize-ability compared to the original scav-class. yippee

## Why It's Good For The Game

the scav-class was an egregious excuse of a ship. this PR will banish it to The Pit.

## Changelog

:cl:
add: the mudskipper-class salvage clipper, the scavenging successor to the scav.
del: removed the scav-class
/:cl:

### CODE REVIEW

File: _maps/configs/independent_scav.json
Comment (Erikafox) [inline _maps/configs/independent_scav.json:1]: just remove it

**Review by Erikafox — DISMISSED** [review]
image
chemical lasers 2030
StrongDMM_jVZ5aLIyj6
use some shitty retro consoles
StrongDMM_gIlduc7Zgi
enshitten it with ration packs
StrongDMM_ZchHtF2h2m
give them the burnt no smoking thing 
StrongDMM_cnfeaeRGxC
This should be a potted plant
image
this sprite only has 1 dir just so you know
@Imaginos16 come hither cruft boy
image
I would give them a connector and a can or two, honestly.

**Comment by goober3** [comment]
> use some shitty retro consoles

when they add connected retro consoles!!

> enshitten it with ration packs

image
i already threw this in here. so the lizard players wouldnt starve

thank yuo for reviewing my PR erika shiptest i will now address your review bye bye

File: _maps/configs/independent_mudskipper.json
Comment (Erikafox) [inline _maps/configs/independent_mudskipper.json:1]: please make their starting funds lower. there should be a few examples in ship configs.

**Review by spockye — COMMENTED** [review]
image

unconnected Scrubber

image

i'd say rotate these. mabye put some radios on the table aswell?

image

dont use that one. it wont store the suit inside automatically. use the Inherit subtype

image

i would say give them a normal energy gun. Scatter is a bit stronk

image

i might be blind, but i dont see any lights in here

**Comment by spockye** [comment]
nevermind, im blind

**Comment by goober3** [comment]
> i'd say rotate these. mabye put some radios on the table aswell?

i rotated the cryo pods in a way that someone could crawl out of them easily & unobstructed. sideways is best, i think.

> i would say give them a normal energy gun. Scatter is a bit stronk

i've tested this and i would call it a sidegrade. the scatter gun deals more damage at the cost of draining faster. and the captain gets it cause they're special-er than everyone else.

**Comment by Xylarr** [comment]
Cryo room has no cryo overseer thing and as a salvage ship it could use a pickaxe or two

**Comment by goober3** [comment]
whdhsadufdfhghf

**Pull request merged**